app/helpers/date_helper.rb

```diff
@@ -3,8 +3,8 @@ def format_date_of_birth(date)
     I18n.l(date.to_date, format: :date_of_birth)
   end
 
-  def format_date_of_visit(date)
-    I18n.l(date.to_date, format: :date_of_visit)
+  def format_near_date(date)
```

Comment:
Near future (or perhaps recent past). It's used for the renewal and expiry of visiting allowance as well as visits.